### PR TITLE
add boolean values to sqlite3 returns

### PR DESCRIPTION
**Pull request by fentie**
Fixes https://youtrack.jetbrains.com/issue/WI-60012

### CODE REVIEW

File: sqlite3/sqlite3.php
Comment (williamdes) [inline sqlite3/sqlite3.php:181]:
```suggestion
     * @return SQLite3Result|true an <b>SQLite3Result</b> object if the query returns results. Otherwise,
```

**Review by williamdes — APPROVED** [review]

**Comment by fentie** [comment]
So what's the next step here? Is someone with write access going to merge this?

File: sqlite3/sqlite3.php
Comment (isfedorov) [inline sqlite3/sqlite3.php:181]: `bool` can be replaced with `false` here as according to  https://github.com/php/php-src/blob/master/ext/sqlite3/sqlite3.stub.php#L60 and https://github.com/php/php-src/blob/master/ext/sqlite3/sqlite3.c#L541 seems method can return `SQLite3Result|false` but can not return `true`

**Comment by isfedorov** [comment]
Hi @fentie 
>So what's the next step here? Is someone with write access going to merge this?

Thank you for the contribution and sorry for the long response. It was vacation time :-) I've left a minor suggestion in the PR. Other than that PR is good to go and after the suggested fix will be merged to master.

**Pull request merged**
